examples/node/package.json

```diff
@@ -17,7 +17,6 @@
     "ts-protoc-gen": "^0.14.0",
     "typescript": "^5.2.2"
   },
-  "type": "module",
```

Comment:
This field enforces ESM for Node.js projects, which is not currently possible with the TS SDK, so removing this to run the project as CommonJS.

Reply:
same for node 20